suite/tests/client-interface/avx512ctx.c

```diff
@@ -0,0 +1,309 @@
+        vmovups  zmm13, [REG_XCX + 64*13]
+        vmovups  zmm14, [REG_XCX + 64*14]
+        vmovups  zmm15, [REG_XCX + 64*15]
+        vmovups  zmm16, [REG_XCX + 64*16]
+        vmovups  zmm17, [REG_XCX + 64*17]
+        vmovups  zmm18, [REG_XCX + 64*18]
+        vmovups  zmm19, [REG_XCX + 64*19]
+        vmovups  zmm20, [REG_XCX + 64*20]
+        vmovups  zmm21, [REG_XCX + 64*21]
+        vmovups  zmm22, [REG_XCX + 64*22]
+        vmovups  zmm23, [REG_XCX + 64*23]
+        vmovups  zmm24, [REG_XCX + 64*24]
+        vmovups  zmm25, [REG_XCX + 64*25]
+        vmovups  zmm26, [REG_XCX + 64*26]
+        vmovups  zmm27, [REG_XCX + 64*27]
+        vmovups  zmm28, [REG_XCX + 64*28]
+        vmovups  zmm29, [REG_XCX + 64*29]
+        vmovups  zmm30, [REG_XCX + 64*30]
+        vmovups  zmm31, [REG_XCX + 64*31]
+#endif
+
+        add      REG_XSP, FRAME_PADDING
+        POP_CALLEE_SAVED_REGS()
+        ret
+        END_FUNC(FUNCNAME)
+#undef FUNCNAME
+
+#define FUNCNAME get_zmm
+        DECLARE_FUNC_SEH(FUNCNAME)
+GLOBAL_LABEL(FUNCNAME:)
+        mov      REG_XCX, ARG1
+        PUSH_CALLEE_SAVED_REGS()
+        sub      REG_XSP, FRAME_PADDING
+        END_PROLOG
+
+       vmovups  [REG_XCX],         zmm0
+       vmovups  [REG_XCX + 64*1],  zmm1
+       vmovups  [REG_XCX + 64*2],  zmm2
+       vmovups  [REG_XCX + 64*3],  zmm3
+       vmovups  [REG_XCX + 64*4], zmm4
+       vmovups  [REG_XCX + 64*5], zmm5
+       vmovups  [REG_XCX + 64*6], zmm6
+       vmovups  [REG_XCX + 64*7], zmm7
+#ifdef X64
+       vmovups  [REG_XCX + 64*8], zmm8
+       vmovups  [REG_XCX + 64*9], zmm9
+       vmovups  [REG_XCX + 64*10], zmm10
+       vmovups  [REG_XCX + 64*11], zmm11
+       vmovups  [REG_XCX + 64*12], zmm12
+       vmovups  [REG_XCX + 64*13], zmm13
+       vmovups  [REG_XCX + 64*14], zmm14
+       vmovups  [REG_XCX + 64*15], zmm15
+       vmovups  [REG_XCX + 64*16], zmm16
+       vmovups  [REG_XCX + 64*17], zmm17
+       vmovups  [REG_XCX + 64*18], zmm18
+       vmovups  [REG_XCX + 64*19], zmm19
+       vmovups  [REG_XCX + 64*20], zmm20
+       vmovups  [REG_XCX + 64*21], zmm12
+       vmovups  [REG_XCX + 64*22], zmm22
+       vmovups  [REG_XCX + 64*23], zmm23
+       vmovups  [REG_XCX + 64*24], zmm24
+       vmovups  [REG_XCX + 64*25], zmm25
+       vmovups  [REG_XCX + 64*26], zmm26
+       vmovups  [REG_XCX + 64*27], zmm27
+       vmovups  [REG_XCX + 64*28], zmm28
+       vmovups  [REG_XCX + 64*29], zmm29
+       vmovups  [REG_XCX + 64*30], zmm30
+       vmovups  [REG_XCX + 64*31], zmm31
+#endif
+
+        add      REG_XSP, FRAME_PADDING
+        POP_CALLEE_SAVED_REGS()
+        ret
+        END_FUNC(FUNCNAME)
+#undef FUNCNAME
+
+#define FUNCNAME init_opmask
+        DECLARE_FUNC_SEH(FUNCNAME)
+GLOBAL_LABEL(FUNCNAME:)
+        mov      REG_XCX, ARG1
+        PUSH_CALLEE_SAVED_REGS()
+        sub      REG_XSP, FRAME_PADDING
+        END_PROLOG
+
+        mov      ax, WORD [REG_XCX]
+        kmovw    k0, eax
+        mov      ax, WORD [REG_XCX + 0x2]
```

Comment:
I can change this into 2*N to be consistent..

Reply:
Yes, in fact you marked my request to do this as "Done" below but it was not done yet...

Reply:
Done.